tests/test_schemas.py

```diff
@@ -951,3 +953,36 @@ def custom_action(self, request, pk):
 
     assert inspector.should_include_endpoint(path, callback)
     assert inspector.get_allowed_methods(callback) == ["GET"]
+
+
+@unittest.skipUnless(coreapi, 'coreapi is not installed')
```

Comment:
The tests should probably use a builtin filter backend (eg, search or ordering), or skip the test if django_filters is not installed (especially since it's not a direct test dependency anymore).

Reply:
Fair enough. I sort of assumed it only breaks with DF, as described in the issue. I'll change that.